.auri/$usr71dkt.md

```diff
@@ -0,0 +1,6 @@
+---
+package: "lucia"
+type: "patch"
+---
+
+Adds a default path of `/` to cookies set with the SvelteKit middleware.
```

Comment:
This should be updated to something along the line of "Update SvelteKit middleware to be compatible with SvelteKit v2"